### PR TITLE
Small CSS changes and tried backend, but not done yet

### PR DESCRIPTION
**Pull request by a5h101**

### CODE REVIEW

File: Rentit-frontend/src/pages/login/login.jsx
Comment (aditya-190) [inline Rentit-frontend/src/pages/login/login.jsx:39]: Please add these values to your .env files.

File: Rentit-frontend/src/pages/login/login.jsx
Comment (aditya-190) [inline Rentit-frontend/src/pages/login/login.jsx:45]: Also remove all the consoles before merging the changes.

File: Rentit-frontend/src/pages/profile/profile.jsx
Comment (aditya-190) [inline Rentit-frontend/src/pages/profile/profile.jsx:17]: Pls Follow the General Naming convection's throughout your web app. For more details, check this out -> https://climbtheladder.com/10-react-naming-conventions-best-practices/

File: Rentit-frontend/src/pages/testPage/test.jsx
Comment (aditya-190) [inline Rentit-frontend/src/pages/testPage/test.jsx:1]: If you are not using this component anymore, this can be removed.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please make required changes before merging the changes.

File: Rentit-frontend/src/pages/login/login.jsx
Comment (a5h101) [reply Rentit-frontend/src/pages/login/login.jsx:39]: Ok

File: Rentit-frontend/src/pages/login/login.jsx
Comment (a5h101) [reply Rentit-frontend/src/pages/login/login.jsx:45]: ok

**Comment by aditya-190** [comment]
Hey you should not merge the changes until it is approved. You have to make the changes and re-request to review it again.

**Pull request merged**
